e2e/evm/test.go

```diff
@@ -150,13 +165,24 @@ func (s *IntegrationTestSuite) TestErc721Deposit() {
 	_, err = erc721Contract2.Owner(tokenId)
 	s.Error(err)
 
-	_, err = bridgeContract1.Erc721Deposit(
-		tokenId, metadata, dstAddr, s.erc721RID, 2, transactor.TransactOptions{},
+	depositTxHash, err := bridgeContract1.Erc721Deposit(
+		tokenId, metadata, dstAddr, s.erc721RID, 2, transactor.TransactOptions{
+			Priority: uint8(2), // fast
+		},
 	)
 	s.Nil(err)
 
+	depositTx, _, err := s.client2.TransactionByHash(context.Background(), *depositTxHash)
+	if err != nil {
+		return
+	}
+	s.Nil(err)
+	// check gas price of deposit tx - 140 gwei
+	s.Equal([]*big.Int{big.NewInt(140000000000)}, depositTx.GasPrice())
```

Comment:
Might as well use default one in erc721 and slow in generic so we test that all cases work.